packages/rich-text/src/to-tree.js

```diff
@@ -9,21 +9,14 @@ import {
 	ZERO_WIDTH_NO_BREAK_SPACE,
 } from './special-characters';
 
-function fromFormat( { type, attributes, object } ) {
+function fromFormat( { type, attributes, unregisteredAttributes, object } ) {
 	const formatType = getFormatType( type );
 
 	if ( ! formatType ) {
 		return { type, attributes, object };
 	}
 
-	if ( ! attributes ) {
-		return {
-			type: formatType.match.tagName,
-			object: formatType.object,
-		};
-	}
-
-	const elementAttributes = {};
+	const elementAttributes = unregisteredAttributes ? { ...unregisteredAttributes } : {};
```

Comment:
Skip it! I'm guessing, but it might also work as:
```js
const elementAttributes = { ...unregisteredAttributes }
```
I tried with
```js
{ ...null } // {}
{ ...undefined } // {}
```
Just sharing how JS tries to be smart :)